api/store/mongo/migrations/migration_23_test.go

```diff
@@ -0,0 +1,160 @@
+package migrations
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/shellhub-io/shellhub/api/pkg/dbtest"
+	"github.com/shellhub-io/shellhub/pkg/models"
+	migrate "github.com/xakep666/mongo-migrate"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func TestMigration23(t *testing.T) {
+	db := dbtest.DBServer{}
+	defer db.Stop()
+
+	migrations := []migrate.Migration{
+		migration_1,
+		migration_2,
+		migration_3,
+		migration_4,
+		migration_5,
+		migration_6,
+		migration_7,
+		migration_8,
+		migration_9,
+		migration_10,
+		migration_11,
+		migration_12,
+		migration_13,
+		migration_14,
+		migration_15,
+		migration_16,
+		migration_17,
+		migration_18,
+		migration_19,
+		migration_20,
+		migration_21,
+		migration_22,
+	}
```

Comment:
I believe every migration test will require this very same logic. We might try to come up with would be a method such as: `migration.GetMigrationsUntil(23)` and `migration.GetNextMigration()` to avoid this duplication in every migration test.

How does it sound?

Reply:
https://play.golang.org/p/EChLMNf_RrP

Reply:
I thought about something like: https://play.golang.org/p/5G6lPgdAwnk

@gustavosbarreto  what do you think?